### ISSUE TITLE
Miscalculated network values for some plays

### ISSUE THREAD

**Issue by lehkost** [dracor-org/dracor-metrics#8]
I noticed this when analysing the current metadata file for GerDraCor (https://dracor.org/api/corpus/ger/metadata).

Two plays have a miscalculated degree (when this value should always be between 0 and 1), plus other network-related miscaculations, so this all seems connected:

![miscalculated-density](https://user-images.githubusercontent.com/6539515/107152778-0ff38700-696a-11eb-8a11-a43f205bad4e.png)

The two erroneous plays are:
* Collin: Coriolan (https://dracor.org/id/ger000474)
* Malß: Der alte Bürger-Capitain oder Die Entführung (https://dracor.org/id/ger000018)

It is unclear why, how and since when these miscalculations occurred and if they can be fixed by a mere recalculation.

Interestingly, calculations as shown on the website (when clicking the above links) seem okay (0.43 for "Coriolan" and 0.57 for "Der alte Bürger-Capitain").

**Comment by cmil** [comment]
I cannot reproduce this. If I follow the links to the plays I see a density o 0.57 for `ger000018` and 0.43 for `ger000474`. The https://dracor.org/api/corpora/ger/metadata endpoint also yields these values for me in both the JSON and CSV representations. From the table shown in the screenshot I would suspect that this is an old version of the data as it is missing the columns `firstAuthor` and `numOfCoAuthors` we added recently.

**Comment by cmil** [comment]
On a closer look, `575` and `425` are the decimal places of the actual density values provided in the meta data. When opening the CSV in macOS' _Numbers_, I noticed that it also interpreted the density values of the two rows in question as those integers `575` and `425` instead of `0.575` and `0.425`. This was fixed after I explicitly specified the decimal point character in the import settings. 

So it looks like spreadsheet applications tend to have a problem correctly guessing the type of the density values for those two rows. The question would then be why.

**Comment by lehkost** [comment]
First, the data is the most recent, I just deleted some columns in the screenshot above so that the table doesn't get too wide (should have mentioned that ;).

Then, the error seems to occur only with numbers that have exactly three digits after the decimal point.

I just tried to reproduce this in LibreOffice Calc and the error is now gone, sort of magically, because I didn't change anything in the setup.
